### PR TITLE
Upgrade to version 19.0.3

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *New version upstream*

## Solution
- *Upgrade to version 19.0.3*
- *Cleanup backup script*
- *Typos*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** : 
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/nextcloud_ynh%20PR333%20(ericgaspar)/)

### CODE REVIEW

File: check_process [hunk 1]
```diff
@@ -12,7 +12,7 @@
 		setup_private=0
 		setup_public=0
 		upgrade=1
-		upgrade=1	from_commit=9c57c2f78f3da979d587d708eb9379ffaa41bcfa
+		upgrade=1	from_commit=af36640ee5a5f1fab69651d638d23ebe47198152
```
Comment (kay0u) [inline check_process:15]: Why did you change this commit?

Comment (ericgaspar) [reply check_process:15]: The Ci fails when upgrading from this commit and the test is ignored... My idea was to test the upgrade from 18.0.7 to 19.0.3 but I can revert back to previous state if you think its better

Comment (kay0u) [reply check_process:15]: Hmm, the problem is that php-mcrypt does not exist. This commit should be the oldest one that works: (not tested)
https://github.com/YunoHost-Apps/nextcloud_ynh/pull/253/commits/c864e086625363d91dde6ba78a652011a991a027

Comment (ericgaspar) [reply check_process]: c864e08 tested and OK

**Review by kay0u — APPROVED** [review]
LGTM & code review

**Review by kay0u — APPROVED** [review]
LGTM & code review, merging in 3 days
